src/imagewriter.cpp

```diff
@@ -502,16 +502,16 @@ bool ImageWriter::getHWFilterListInclusive() {
     return _deviceFilterIsInclusive;
 }
 
-bool ImageWriter::andSupportedFeatures(const QString &feature) {
-    return this->checkHWFeatureSupport(feature) && this->checkSWFeatureSupport(feature);
+bool ImageWriter::andCapabilities(const QString &cap) {
+    return this->checkHWCapability(cap) && this->checkSWCapability(cap);
```

Comment:
1. Fixed
2. I now added a second optional parameter that would allow to specify different naming and still has the option to only specifying a unified naming.